test/suites/smoke-test/test-invulnerables-priority.ts

```diff
@@ -0,0 +1,63 @@
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import { ApiPromise } from "@polkadot/api";
+
+describeSuite({
+    id: "S06",
+    title: "Sample suite that only runs on Dancebox chains",
+    foundationMethods: "read_only",
+    testCases: ({ it, context }) => {
+        let api: ApiPromise;
+        let runtimeVersion;
+
+        beforeAll(async () => {
+            api = context.polkadotJs();
+            runtimeVersion = api.runtimeVersion.specVersion.toNumber();
+        });
+
+        it({
+            id: "C01",
+            title: "Invulnerables have priority over staking candidates",
+            test: async function () {
+                if (runtimeVersion < 200) {
+                    return;
+                }
+
+                // TODO: we should read the invulnerables at the start of this session, because that's when collators are updated
```

Comment:
Any tips for this?

Reply:
I think its alright. This tests run every hour so you will at least catch once the changes per session.

Maybe we can have another test for checking that at session changes, we have the event of selecting new candidates